packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -410,6 +410,11 @@ export const codeTransformTroubleshootDownloadError =
 export const codeTransformPrereqDoc =
     'https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html#prerequisites'
 
+export const codeTransformBillingText =
```

Comment:
Why don't you make this a function that takes in the number of lines of code and then does the calculations inline using string interpolation?

e.g.

```typescript
export const codeTransformBillingText = (linesOfCode: number) {
    return `<p>${linesOfCode} lines of code were submitted for transformation. If you reach the quota for lines of code included in your subscription, you will be charged ${CodeWhispererConstants.codeTransformBillingRate.toString()} for each additional line of code. You might be charged up to ${(linesOfCode * CodeWhispererConstants.codeTransformBillingRate).toFixed(2)} for this transformation. To avoid being charged, stop the transformation job before it completes. For more information on pricing and quotas, see [Amazon Q Developer pricing](https://aws.amazon.com/q/developer/pricing/).</p>`
}
```

Reply:
Fixed